apps/web/src/components/layout/components/SideNav.tsx

```diff
@@ -15,11 +15,9 @@ import { colors, NavMenu, SegmentedControl, shadows } from '../../../design-syst
 import { Activity, Bolt, Box, Settings, Team, Repeat, CheckCircleOutlined, Brand } from '../../../design-system/icons';
 import { ChangesCountBadge } from './ChangesCountBadge';
 import { useEnvController } from '../../../hooks';
-import { useAuthContext } from '../../providers/AuthProvider';
 import OrganizationSelect from './OrganizationSelect';
 import { useSpotlightContext } from '../../providers/SpotlightProvider';
 import { HEADER_HEIGHT } from '../constants';
-import { LimitBar } from '../../../pages/integrations/components/LimitBar';
```

Comment:
through some files, I've removed the unused imports

Reply:
🧹